task_opts.go

```diff
@@ -51,3 +51,15 @@ func WithProcessKill(ctx context.Context, p Process) error {
 	<-s
 	return nil
 }
+
+type KillInfo struct {
+	// All kills all processes inside the task
```

Comment:
Probably comment that this only works for task. IIUC, `exec.KIll` will just ignore this field inside the shim.

Reply:
done